lib/srv/authhandlers.go

```diff
@@ -521,14 +553,35 @@ func (h *AuthHandlers) IsHostAuthority(cert ssh.PublicKey, address string) bool
 	return true
 }
 
+// loginChecker checks if the Teleport user should be able to login to
+// a node. It is created for tests.
```

Comment:
This interface is extracted to enable testing, but it's not created only for testing. 
```suggestion
// a node.
```

Reply:
Done